src/utils/convertPipes/index.js

```diff
@@ -129,9 +138,10 @@ const getPipedData = store => (element, ctx) => {
   let placeholder = {};
 
   if (transformed) {
-    let dateFormat;
+    let dateFormat, unitType;
     if (entity.properties) {
       dateFormat = entity.properties.format;
+      unitType = entity.properties.unit;
```

Comment:
Would it be worth combining these? So we only assign one variable and then pass that variable through the options function.

Basically check if format exists and then assign that to like a properties variable. Then push the property value into the function and replace dateFormat and unitType with property?

Does that make any sense?

Reply:
That makes sense, I'm not entirely sure on if we should though; it would work but moving forward I'm not sure if we'll remember why we did it or how it works. `unitType` and `dateFormat` have different behaviour, too, when they're used elsewhere.